CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 * Rust
   * Introduce the Counter metric type in the RLB.
   * Introduce the String metric type in the RLB.
+  * Introduce the UUID metric type in the RLB.
```

Comment:
@chutten: Oops, I think this is in the wrong changelog section...

Reply:
Oh bother.

Reply:
https://github.com/mozilla/glean/pull/1333